src/poker/game/ActionHandler.java

```diff
@@ -32,6 +35,11 @@ public HandleActionResult handleAction(PossibleAction lastAction, Player activeP
         } else if (lastAction == PossibleAction.CALL) {
             boolean isAllin = handleCall(activePlayer, nonActivePlayer);
             result.setAllIn(isAllin);
+            if (isAllin){
+                printEquity(gameState, activePlayer, nonActivePlayer, deck);
+                gameState.setGamePhase(GamePhase.RIVER);
+                handleNextStreet(gameState,activePlayer,nonActivePlayer);
```

Comment:
pisalem ze to ma byc w funkcji handleAllIn() ale zapomnialem ze mamy juz taka, to wez zrobi funkcje handleCallAllIn